preposterous-kumquat/preposterous-kumquat
# Pipe POST /upload to photoProcessor & Save photo data to db

#### What's this PR do?

Pipe's POST /upload to photoProcessor, with unique photo id generated from db. Upon completion of photo processing from photoProcessor will store theme, url, gps data to db. Also creates join table between keywords and themes.
Create config/config.js file for dev/prod ports
#### Where should the reviewer start?

Launch main server, photoprocesser server and psql
#### How should this be manually tested?

Using postman to post to localhost:3000/upload with photo and them in the form data, see screenshot
#### Any background context you want to provide?
#### What are the relevant tickets?
#### Screenshots (if appropriate)

<img width="1280" alt="screen shot 2016-09-28 at 9 32 37 pm" src="https://cloud.githubusercontent.com/assets/8231534/18941123/35e5eabe-85c3-11e6-93bf-fe79786c9ca5.png">
#### Questions:
